source/git-api.js

```diff
@@ -401,11 +404,29 @@ exports.registerApi = (env) => {
       .finally(emitGitDirectoryChanged.bind(null, req.query.path));
   });
 
+  const createBranchIfPossible = (branchName, path, retry) => {
+    return gitPromise(['branch', branchName], path).catch((e) => {
+        if (retry > -1) {
```

Comment:
Just to be sure this is intended: this will retry `4` times. 3, 2, 1, 0

Reply:
On grand scheme of things... Meh... but yeah will fix it for the consistency sake.